test/transaction.model.test.js

```diff
@@ -98,4 +119,23 @@ describe('transaction model', () => {
     })
     .catch(done);
   });
+
+  it('createFromPayload() generates a new activity', done => {
+
+    beforeEach(() => sinon.spy(Transaction, 'createActivity'));
+
+    afterEach(() => Transaction.createActivity.restore());
```

Comment:
This errors out `TypeError: Transaction.createActivity.restore is not a function`